lib/AlipayAccount.php

```diff
@@ -47,8 +47,9 @@ public function instanceUrl()
      */
     public static function retrieve($_id, $_opts = null)
     {
-        $msg = "Alipay accounts cannot be accessed without a customer ID. " .
-               "Retrieve an Alipay account using \$customer->sources->retrieve('alipay_account_id') instead.";
+        $msg = "Alipay accounts cannot be retrieved without a customer ID. " .
+               "Retrieve an Alipay account using `Customer::retrieveSource(" .
+               "'customer_id', 'alipay_account_id')`.";
```

Comment:
technically we should say "This is deprecated please go away"

Reply:
but what if someone wants to retrieve an Alipay account from 5 years ago